maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -1142,8 +1142,8 @@ private void warnIfForkCountIsZero()
     {
         if ( "0".equals( getForkCount() ) )
         {
-            getConsoleLogger().warning( "The parameter forkCount should likely not be 0, not forking a JVM for tests "
-                + "reduce test accuracy, ensure to have a <forkCount> >= 1." );
+            getConsoleLogger().warning( "The parameter forkCount should likely not be 0. Not forking a JVM for tests "
+                + "reduces test accuracy. Ensure to have a <forkCount> >= 1." );
```

Comment:
Technically it is correct I think, but personally I would rewrite the middle sentence in the other way around. What do you think of: `Forking a JVM for tests improves test accuracy.`

Reply:
Feel free to do so. As the maintainer - you can push to my branch. You can do separate PR if you prefer and close this one without merge.

Reply:
Sure thing, but I am just reviewing your PR, which you asked for. Thanks for fixing the grammar, it will definitely make it more clear. If you won't change it, I can do it tomorrow. I currently do not have the time for it.

Reply:
Thanks for checking - sure thing.
I was only bothered by the verb in first place. I have no opinion regarding middle sentence.
I just had to retrigger Travis twice (once for failing _GitHub CI for *Nix / build (macOS-latest)_, once for _GitHub CI for *Nix / build (ubuntu-latest)_) without changing my changes (one can check diffs for my forced pushes). Only then I marked this PR as ready for review and asked for one.
I have no intention to play with Travis again.

Reply:
@pzygielo 
This is still only one file of change?
Pls squash your commits to one and rebase it.

Reply:
Thanks for checking.
As there is no conflict - I'm not going to rebase.
As you by the rule do not create merge commits in this project - you can and do squash on merge - I'm not going to do it.
Instead I hope Martin's change will cover the case.
Have a nice day.